src/pages/EmergencyViewPage/EmergencyViewPage.tsx

```diff
@@ -121,16 +117,17 @@ export const EmergencyViewPage: React.FC = () => {
   }
 
   return (
-    <WithNavigationBar withSchedule={false}>
-      <div
-        className="EmergencyView__banner"
-        style={{
-          backgroundImage: `url(${validBannerImageUrl})`,
-        }}
-      ></div>
+    <WithNavigationBar withSchedule={false} hasBackButton={false}>
       <div className="EmergencyView">
         <EmergencyViewTabs updateTab={updateTab} selectedTab={selectedTab} />
-        <div className="EmergencyView_main">
+        <div
+          className="EmergencyView__main"
+          style={{
+            backgroundImage: `url(${validBannerImageUrl})`,
+            backgroundSize: "cover",
+            backgroundRepeat: "no-repeat",
+          }}
+        >
```

Comment:
Style prop not needed here, can provide backup background if necessary in the `EmergencyViewPage.scss` and move the static properties there as well.

The inline style could probably be handled better with `useCss()`, example in https://github.com/sparkletown/sparkle/blob/staging/src/components/molecules/ScheduleVenueDescription/ScheduleVenueDescription.tsx#L30-L37

Reply:
removed inline style prop 6eb97d6